From a48a36bf3e1ad8c7bc8ee25a699dad512eeab14e Mon Sep 17 00:00:00 2001
From: =?utf-8?q?Diego=20Elio=20'Flameeyes'=20Petten=C3=B2?= <flameeyes@gmail.com>
Date: Thu, 21 May 2009 12:30:42 +0200
Subject: [PATCH 2/2] Also fix the configure for hdspconf for LIBS/LDFLAGS mistakes.

Commit 56970e8143b4d171a118d114b1ddfa7621401127 already took care of this
for the other tools, but hdspconf somewhat was excluded, fix this now.
---
 hdspconf/configure.in |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/hdspconf/configure.in b/hdspconf/configure.in
index 1aa6feb..a32ff5f 100644
--- a/hdspconf/configure.in
+++ b/hdspconf/configure.in
@@ -36,6 +36,6 @@ FLTK_CXXFLAGS="`$FLTK_CONFIG $fltk_config_args --cxxflags` -fno-exceptions"
 FLTK_LIBS=`$FLTK_CONFIG $fltk_config_args --ldflags`
 
 CXXFLAGS="$CXXFLAGS $ALSA_CFLAGS $FLTK_CXXFLAGS"
-LDFLAGS="$LDFLAGS $ALSA_LIBS $FLTK_LIBS"
+LIBS="$LIBS $ALSA_LIBS $FLTK_LIBS"
 
 AC_OUTPUT(Makefile src/Makefile pixmaps/Makefile)
-- 
1.6.3.1

